tests/visual/scenes/graphics/graphics-pixel-line.scene.ts

```diff
@@ -0,0 +1,24 @@
+import { Graphics } from '../../../../src/scene/graphics/shared/Graphics';
+
+import type { Renderer } from '../../../../src/rendering/renderers/types';
+import type { Container } from '../../../../src/scene/container/Container';
+import type { TestScene } from '../../types';
+
+export const scene: TestScene = {
+    it: 'should render a pixel line correctly',
+    create: async (scene: Container, _renderer: Renderer) =>
+    {
+        const graphics = new Graphics();
+
+        graphics
+            .moveTo(8, 0).lineTo(0, 8).moveTo(12, 20).lineTo(20, 12)
+            .stroke({ color: 0, pixelLine: true })
+            .rect(128 / 4, 128 / 4, 128 / 2, 128 / 2)
```

Comment:
I notice the stroke on the rect doesn't close, right? The upper left corner looks open.

Reply:
yep - that correct. they should appear as two dashes in the top left